src/App.tsx

```diff
@@ -116,26 +119,38 @@ function App() {
         } else {
           // If any errors occurred, report them
           response.data.errors.forEach((error) => {
-            enqueueSnackbar(`Failed to retrieve ${NBResource} options from ${error.node_name}`, {
-              variant: 'warning',
-              action,
-            });
+            setNotifications((prev) => [
+              ...prev,
+              {
+                id: prev.length + 1,
```

Comment:
I don't think this is a safe way of setting the `id`, let's use a safer approach like UUID.